### ISSUE TITLE
Managed Server Unable to connect to the Administration Server

### ISSUE THREAD

**Issue by amyles** [oracle/weblogic-kubernetes-operator#547]
Hi 
I've created a WLS domain on my kubernetes cluster, all pods are running. The admin server has started fine and I can access the admin console. However the managed servers have not registered with the admin server. I see this error in the logs:

<Dec 3, 2018 10:46:07 AM GMT> <Info> <Management> <BEA-141307> <Unable to connect to the Administration Server. Waiting 5 second(s) to retry (attempt number 3 of 3).> 
<Dec 3, 2018 10:50:33 AM GMT> <Info> <Management> <BEA-141298> <Could not register with the Administration Server: java.rmi.RemoteException: 
 [from AdminURL 0] weblogic.deploy.service.internal.transport.UnreachableHostException: [Deployer:149145]Unable to contact "null". Deployment is deferred until "null" becomes available. WARNING: If this is a redeploy app and contains JSP pages that have been touched, these JSP changes wont be reflected when the server starts unless precompile is set to true in weblogic.xml; nested exception is: 
	java.net.ConnectException: Tried all: '1' addresses, but could not connect over HTTP to server: 'domain1-admin-server', port: '7001'
 failed reasons:
  [0] address:'domain1-admin-server/10.96.236.22',port:'7001' : java.net.ConnectException: Connection timed out (Connection timed out)
 [from AdminURL 1] weblogic.deploy.service.internal.transport.UnreachableHostException: [Deployer:149145]Unable to contact "null". Deployment is deferred until "null" becomes available. WARNING: If this is a redeploy app and contains JSP pages that have been touched, these JSP changes wont be reflected when the server starts unless precompile is set to true in weblogic.xml; nested exception is: 
	java.net.ConnectException: Tried all: '1' addresses, but could not connect over HTTP to server: '10.96.236.22', port: '7001'
 failed reasons:
  [0] address:'/10.96.236.22',port:'7001' : java.net.ConnectException: Connection timed out (Connection timed out)
> 

10.96.236.22 is the cluster IP of the admin server pod and the admin server is listening on 7001. When I exec to the admin pod I can curl http://10.96.236.22:7001/console OK, when I try on a managed server pod it times out.

**Comment by amyles** [comment]
Hi @mriccell 
The admin server is exposed fined via the adminNodePort, I can access it from my desktop with no issues. 

What I see though is that only the admin server has started, the two managed servers are down. When I exec to the managed server pods I can see that the managed server is trying to start but can't access the admin server to register (as per the log above). AFAIK the managed server pod should be able to access the admin server pod via the cluster IP. 
I am running a three node OKE cluster on OCI.

**Comment by amyles** [comment]
Hi @mriccell 
I am running this on OKE on Oracle Cloud Infrastructure and it seems that an additional ingress rule is required above those created on cluster creation to allow node to node traffic. 
I added a stateful ingress rule in my VCN for 10.0.0.0/16 for all protocols (10.0.0.0/16 is the VCN CIDR) and the issue was fixed. The WLS console reported all the managed servers were up and I could curl the admin server URL from the managed server pod as expected. Removing the ingress rule causes the issue to return.
